tests/test_dist_packages_versions.py

```diff
@@ -37,6 +37,9 @@ def test_versions(
     mender_configure_version,
     mender_dist_packages_versions,
 ):
+    verify_package_version(
+        mender_version, mender_dist_packages_versions["mender-client"]
+    )
     verify_package_version(
         mender_version, mender_dist_packages_versions["mender-client"]
     )
```

Comment:
This new test is not adding any value. I don't think it is important to explicitly verify the `-dev` package and we can drop it. Similarly mender-configure generates more than one package and we only verify the version of the base one.